distribution/docs/src/main/resources/content/_managing/_running/solr-service.adoc

```diff
@@ -7,75 +7,4 @@
 
 ==== {title}
 
-These instructions are for configuring Solr as a service managed by the operating system.
-
-===== Configure Solr as a Windows Service
-
-Windows users can use the *Task Scheduler* to start Solr as a background process.
-
-. If ${branding} is running, stop it.
-. Edit `${home_directory}/etc/custom.system.properties` and set `start.solr=false`.
-  This prevents the ${branding} scripts from attempting to manage Solr's lifecycle.
-. Start the *Windows Task Scheduler* and open the *Task Scheduler Library*.
-. Under the Actions pane, select *Create Basic Task...*.
-. Provide a useful name and description, then select *Next*.
-. Select *When the computer starts* as the Trigger and select *Next*.
-. Select *Start a program* as the Action and select *Next*.
-. Select the script to start Solr:
-+
------
-${home_directory}\bin\ddfsolr.bat
------
-+
-. Add the argument `start` in the window pane and select *Next*.
-. Review the settings and select *Finish*.
-
-It may be necessary to update the *Security Options* under the task *Properties* to
-*Run with highest privileges* or setting user to "SYSTEM".
-
-Additionally, the process can be set to restart if it fails. The option can be found in the
- the *Properties* > *Settings* tab.
-
-Depending on the system it may also make sense to delay the process from starting for a
-few minutes until the machine has fully booted. Open the task's *Properties* settings and
-
- .. Select *Triggers*.
- .. Select *Edit*.
- .. Select *Advanced Settings*.
- .. Select *Delay Task*.
-
-
-===== Configure Solr as a Systemd Service
```

Comment:
:exclamation: These instruction are removed, but the solr.service file still exists.

Reply:
I almost removed that file.  They should be following the Taking Solr to Production service instructions instead which is linked in the install SolrCloud instructions for the Solr Catalog Provider documentation.  I will remove it since no one should use it without these instructions.

https://lucene.apache.org/solr/guide/7_7/taking-solr-to-production.html